cranelift/codegen/src/isa/aarch64/inst/emit.rs

```diff
@@ -906,6 +906,7 @@ impl MachInstEmit for Inst {
                     ALUOp::OrrNot => 0b001_01010001,
                     ALUOp::EorNot => 0b010_01010001,
                     ALUOp::AndNot => 0b000_01010001,
+                    ALUOp::RotR => 0b000_10011100,
```

Comment:
It looks like (at least by my reading) `rotr` is an alias for / special case of `extr`, not the other way around, so perhaps we could rename the `ALUOp`?